provider/filestore.go

```diff
@@ -270,6 +277,27 @@ func NewS3FileStore(config Config) (FileStore, error) {
 		return nil, wrapped
 	}
 
+	args := []func(*awsv2cfg.LoadOptions) error{
```

Comment:
what is all this new stuff for? so we can connect with retries?

Reply:
just a thought, this looks like code I would copy from here and paste in a scratch file to connect to s3 to test out their api's locally, might be good to have it as a separate function so folks can easily search/use it.

Reply:
I want to move all the connection stuff into one place soon, makes testing easier too